TrajectoryPlannerROS:
        max_vel_x: 0.3
        min_vel_x: 0.05
        max_vel_theta: 0.4
        min_in_place_vel_theta: 0.3
        acc_lim_theta: 1
        acc_lim_x: 1
        acc_lim_y: 1
        holonomic_robot: true
        yaw_goal_tolerance: 0.1
        xy_goal_tolerance: 0.1
